README.md

```diff
@@ -23,25 +106,96 @@ After installing and configuring those, download [docker-compose.yml](https://ra
 docker-compose up -d
 ```
 
+This will build the containers, start the services and run the database migrations. Additionally, it
+creates a user, scope, role and and ACL for administration based on the settings [documented above](#Bootstrapping)
+
+## Getting started
+
 You can now access the api at [http://localhost:8000/api/v1/](http://localhost:8000/api/v1/).
 
-### Configuration
 
-emeis is a [12factor app](https://12factor.net/) which means that configuration is stored in environment variables.
-Different environment variable types are explained at [django-environ](https://github.com/joke2k/django-environ#supported-types).
+## Extension points
 
-#### Common
+Caluma is meant to be used as a service with a clean API hence it doesn't provide a Django app.
```

Comment:
1. Caluma -> Emeis
2. That's not entirely true anymore (both for Caluma and Emeis)

Reply:
Removed the line.